index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>FitNotFat</title>
+</head>
+<body>
+
+    <h1>hello world!</h1>    
+    
+
+    <script src="SrcCompiled/App.js"></script>
```

Comment:
I suggest to import index.ts file, because parcel supports transforming Typescript without any additional configuration ( https://parceljs.org/typeScript.html ).

Reply:
good point, in this case compiling the code with typescript itself doesn't make too much sense at all (we can use parcel for everything)